ios/Classes/core/PHAsset+PHAsset_getTitle.m

```diff
@@ -38,4 +38,114 @@ - (NSString *)title {
     }
 }
 
+- (BOOL)isAdjust {
+  NSArray<PHAssetResource *> *resources =
+      [PHAssetResource assetResourcesForAsset:self];
+  if (resources.count == 1) {
+    return NO;
+  }
+
+  if (self.mediaType == PHAssetMediaTypeImage) {
+    return [self imageIsAdjust:resources];
+  } else if (self.mediaType == PHAssetMediaTypeVideo) {
+    return [self videoIsAdjust:resources];
+  }
+
+  return NO;
+}
+
+- (BOOL)imageIsAdjust:(NSArray<PHAssetResource *> *)resources {
+  int count = 0;
+  for (PHAssetResource *res in resources) {
+    if (res.type == PHAssetResourceTypePhoto ||
+        res.type == PHAssetResourceTypeFullSizePhoto) {
+      count++;
+    }
+  }
+
+  return count > 1;
+}
+
+- (BOOL)videoIsAdjust:(NSArray<PHAssetResource *> *)resources {
+  int count = 0;
+  for (PHAssetResource *res in resources) {
+    if (res.type == PHAssetResourceTypeVideo ||
+        res.type == PHAssetResourceTypeFullSizeVideo) {
+      count++;
+    }
+  }
+
+  return count > 1;
+}
+
+- (PHAssetResource *)getAdjustResource {
+  NSArray<PHAssetResource *> *resources =
+      [PHAssetResource assetResourcesForAsset:self];
+  if (resources.count == 0) {
+    return nil;
+  }
+
+  if (resources.count == 1) {
+    return resources[0];
+  }
+
+  if (![self isAdjust]) {
+    for (PHAssetResource *res in resources) {
+      if (self.mediaType == PHAssetMediaTypeImage ||
+          res.type == PHAssetResourceTypeFullSizePhoto ||
+          res.type == PHAssetResourceTypePhoto) {
+        return res;
+      }
+
+      if (self.mediaType == PHAssetMediaTypeVideo ||
+          res.type == PHAssetResourceTypeFullSizeVideo ||
+          res.type == PHAssetResourceTypeVideo) {
+        return res;
+      }
+    }
+
+    return nil;
+  }
+
+  for (PHAssetResource *res in resources) {
+    if (self.mediaType == PHAssetMediaTypeImage ||
+        res.type == PHAssetResourceTypeFullSizePhoto) {
+      return res;
+    }
+
+    if (self.mediaType == PHAssetMediaTypeVideo ||
+        res.type == PHAssetResourceTypeFullSizeVideo) {
+      return res;
+    }
+  }
```

Comment:
Hey as this is adjusted image shouldn't it be `[resources reverseObjectEnumerator]`?

Reply:
@CaiJingLong yep just checked the `0.6.0-dev.7` version and the issue is still persistent.